### DISCUSSION TITLE
Bot disguised as husk won't use turret to attack husk outside submarine

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14996] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

although to him husk is friendly but it does not mean husk is friendly to submarine or other crew numbers

### Reproduction steps

1. equip a bot with cultist robe
2. order him to operate the turret
3. spawn husk outside submarine
4. the bot will not attack the husk

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.18.1 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by itchyOwl** [comment]
Confirmed the issue. It makes sense that the bots don't attack the husks with small arms, but I guess they should still use turrets to target them. We can't just force compare the teams here, because that'd mean we could shoot at other friendly characters (NPCs or pets).

**Comment by itchyOwl** [comment]
Addressed in https://github.com/FakeFishGames/Barotrauma-development/pull/6179.

**Comment by Jasontti** [comment]
Tested the implemented fix and found no issues.
Closing discussion as resolved.
